ci/platforms/config.hercules

```diff
@@ -4,6 +4,6 @@ export GFS_CI_ROOT=/work2/noaa/stmp/GFS_CI_ROOT/HERCULES
 export ICSDIR_ROOT=/work/noaa/global/glopara/data/ICSDIR
 export STMP="/work2/noaa/stmp/${USER}/HERCULES"
 export PTMP="/work2/noaa/stmp/${USER}/HERCULES"
```

Comment:
These should be removed.

Reply:
Yes, just as where on Orion, not sure how that got missed. Thanks.